client/overview/modal/connection-success/strings.tsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable max-len */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export default {
+	button: __( 'Dismiss', 'woocommerce-payments' ),
+
+	heading: __( "You're ready to accept payments!", 'woocommerce-payments' ),
+
+	description: __(
+		'Great news — your WooPayments account has been activated. You can now start accepting payments on your store.',
```

Comment:
could you please extract WooPayments so it's not translated? see this example https://github.com/Automattic/woocommerce-payments/blob/19f382152d9604fc605bcc5a4c0d9614e42936ab/client/connect-account-page/strings.tsx#L69-L69